libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/EHorizonPosition.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.base.trip.model
+
+/**
+ * The position on the current EHorizon.
+ */
+data class EHorizonPosition(
```

Comment:
Same as above.

Reply:
@mskurydin @SiarheiFedartsou could you help here? Are we envisioning extending this / adding more data as part of `EHorizonPosition` in the future?